packages/scandipwa/src/component/MyAccountMyWishlist/MyAccountMyWishlist.style.scss

```diff
@@ -46,10 +46,21 @@
             max-width: 100%;
             margin: 0 auto;
             margin-block-end: var(--myaccount-wihslist-products-margin-bottom);
+
+            & > .Loader .Loader-Scale {
+                position: fixed;
+                inset-block-start: auto;
+                inset-block-end: 50%;
+                transform: translate(-50%, 50%);
+            }
         }
 
         @include desktop {
             grid-column: 2;
+
+            & > .Loader .Loader-Scale {
+                inset-block-start: 11%;
```

Comment:
I think we cannot rely on such values.
If wishlist card height is constant, then to harcode the top value would be a more reliable approach.
But I personally don't think there should be a loader. Since we cannot center the loader - it will go down when the second row of products will appear. And we cannot rely on that wishlist card will always stay the same height.
@carinadues  Maybe "Add All To Cart" button should have a loading indicator instead?

![image](https://user-images.githubusercontent.com/73945186/162800700-8acec8c9-2960-4cdd-a5bb-85a2fbcee967.png)

Reply:
Lets left like it was on master (before changes in pr for desktop) - after the call with Aleksandra and Levan